src/net_processing.cpp

```diff
@@ -2558,20 +2545,149 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
+        // Must have a version message before anything else
+        LOCK(cs_main);
+        Misbehaving(pfrom->GetId(), 1);
+        return false;
+    }
+
+    // At this point, the outgoing message serialization version can't change.
+    CNetMsgMaker msgMaker(pfrom->GetSendVersion());
+    
+    if (strCommand == NetMsgType::VERACK)
+    {
+        return MsgVerack(pfrom, connman, msgMaker);
+    }
+    else if (strCommand == NetMsgType::ADDR)
+    {
+        return MsgAddr(pfrom, vRecv, connman, interruptMsgProc);
+    }
+    else if (strCommand == NetMsgType::SENDHEADERS)
+    {
+        return MsgSendheaders(pfrom);
+    }
+    else if (strCommand == NetMsgType::SENDCMPCT)
+    {
+        return MsgSendcmpct(pfrom, vRecv);
+    }
+    else if (strCommand == NetMsgType::INV)
+    {
+        return MsgInv(pfrom, vRecv, chainparams.GetConsensus(), connman, msgMaker, interruptMsgProc);
+    }
+    else if (strCommand == NetMsgType::GETDATA)
+    {
+        return MsgGetdata(pfrom, vRecv, chainparams.GetConsensus(), connman, interruptMsgProc);
+    }
+    else if (strCommand == NetMsgType::GETBLOCKS)
+    {
+        return MsgGetblocks(pfrom, vRecv, chainparams.GetConsensus());
+    }
+    else if (strCommand == NetMsgType::GETBLOCKTXN)
+    {
+        return MsgGetblocktxn(pfrom, vRecv, chainparams.GetConsensus(), connman, interruptMsgProc);
+    }
+    else if (strCommand == NetMsgType::GETHEADERS)
+    {
+        return MsgGetheaders(pfrom, vRecv, connman, msgMaker);
+    }
+    else if (strCommand == NetMsgType::TX)
+    {
+        return MsgTx(pfrom, strCommand, vRecv, chainparams, connman, msgMaker);
+    }
+    else if (strCommand == NetMsgType::CMPCTBLOCK && !fImporting && !fReindex) // Ignore blocks received while importing
+    {
+        return MsgCmpctblock(pfrom, vRecv, nTimeReceived, chainparams, connman, interruptMsgProc, msgMaker);
+    }
+    else if (strCommand == NetMsgType::BLOCKTXN && !fImporting && !fReindex) // Ignore blocks received while importing
+    {
+        return MsgBlocktxn(pfrom, vRecv, chainparams, connman, msgMaker);
+    }
+    else if (strCommand == NetMsgType::HEADERS && !fImporting && !fReindex) // Ignore headers received while importing
+    {
+        return MsgHeaders(pfrom, vRecv, chainparams, connman, msgMaker);
+    }
+    else if (strCommand == NetMsgType::BLOCK && !fImporting && !fReindex) // Ignore blocks received while importing
+    {
+        return MsgBlock(pfrom, vRecv, chainparams);
+    }
+    else if (strCommand == NetMsgType::GETADDR)
+    {
+        return MsgGetaddr(pfrom, connman);
+    }
+    else if (strCommand == NetMsgType::MEMPOOL)
+    {
+        return MsgMempool(pfrom, connman);
+    }
+    else if (strCommand == NetMsgType::PING)
+    {
+        return MsgPing(pfrom, vRecv, connman, msgMaker);
+    }
+    else if (strCommand == NetMsgType::PONG)
+    {
+        return MsgPong(pfrom, vRecv, nTimeReceived);
+    }
+    else if (strCommand == NetMsgType::FILTERLOAD)
+    {
```

Comment:
According to our style guideline all those  curly brackets are on the "wrong" line. If we want to stick to the current style guideline, new code should not violate it. If we don't want to stick with the current style guide, we should weaken in with regard to new lines.